extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/dispatcher/atlas/AtlasClientConf.scala

```diff
@@ -21,12 +21,16 @@ import org.apache.atlas.ApplicationProperties
 import org.apache.commons.configuration.Configuration
 import org.apache.spark.kyuubi.lineage.SparkContextHelper
 
+import java.util
+
 class AtlasClientConf(configuration: Configuration) {
 
   def get(entry: ConfigEntry): String = {
     configuration.getProperty(entry.key) match {
       case s: String => s
       case l: List[_] => l.mkString(",")
+      case jl: util.List[_] => val jlString = jl.toString
```

Comment:
seems there is no chance to return a Scala List? can we handle it like

```patch
+ import scala.collection.JavaConverters._
  ...
- case l: List[_] => l.mkString(",")
+ case jl: util.List[_] => val jlString = jl.asScala.mkString(",")
```

Reply:
I thought there would be scala's List configuration, so I didn't delete the original code, and now it doesn't seem to be used. 
I have tested your change in the test environment, and it can run perfectly, so modify it according to your way

Reply:
yea, the altas configuration is a pure Java class, which should not involve Scala collection. could you update and push your change then?

Reply:
@SwordyZhao Thanks for your contribution. Could you update this code if you have time?

Reply:
ok, i have update and push again

Reply:
> ok, i have update and push again

Thanks, please run `dev/reformat` to fix code style.

Reply:
ok, i have run the shell, and push again